.vscode/tasks.json

```diff
@@ -16,6 +16,132 @@
         "isDefault": true
       }
     },
+    {
+      "label": "Build neovim extension only",
+      "dependsOn": [
+        "neovim start",
+        "ESBuild-neovim-registry",
+        "ESBuild-cursorless-neovim"
+      ],
+      "group": "build"
+    },
+    {
+      "label": "Build neovim extension tests only",
+      "dependsOn": [
```

Comment:
I would depend on TSBuild here like we do for vscode tests, as it's preferable to detect type issues before trying to run the tests